src/Kernel32.Desktop/Kernel32+SafeLibraryHandle.cs

```diff
@@ -16,6 +16,8 @@ public static partial class Kernel32
         /// </summary>
         public class SafeLibraryHandle : SafeHandle
         {
+            public static readonly SafeLibraryHandle Null = new SafeLibraryHandle(new IntPtr(0), false);
```

Comment:
I'll fix, I don't have a strong opinion about `IntPtr`, `new Guid()` is evil as it's too easy to confuse with `Guid.NewGuid()` but i'll follow whatever the norm is here for `IntPtr` ;)